lib/pi_piper.rb

```diff
@@ -1,8 +1,39 @@
 require 'eventmachine'
-Dir[File.dirname(__FILE__) + '/pi_piper/*.rb'].each {|file| require file unless file.end_with?('bcm2835.rb') }
+
+require 'pi_piper/version'
+require 'pi_piper/frequency'
+require 'pi_piper/driver'
+require 'pi_piper/i2c'
+require 'pi_piper/pin'
+require 'pi_piper/spi'
+require 'pi_piper/pwm'
 
 module PiPiper
+  class PinError < StandardError; end
```

Comment:
Yeah you're right, this isn't used anywhere anymore. Go ahead and remove it entirely.